<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="0"
            failures="1"
            tests="1">

  <testsuite name="index"
    errors="0" failures="1" hostname="" id="" package="index" skipped="0"
    tests="1" time="13.79" timestamp="Fri, 11 Aug 2017 13:29:09 GMT">
  
    <testcase name="OI Fail Clearance" classname="index" time="13.79" assertions="2">

      <failure message="Login Successfull!">    at login (/Users/warrenlongmire/Sites/argo/OwnersEdge/OwnersEdge-Client/e2e/tests/utils/login.js:9:8)
    at Object.OI Fail Clearance (/Users/warrenlongmire/Sites/argo/OwnersEdge/OwnersEdge-Client/e2e/tests/e2e/index.js:29:5)</failure>

    
    </testcase>
  

  

  
  </testsuite>
</testsuites>
